deploy/charts/app-orch-tenant-controller/values.yaml

```diff
@@ -51,10 +51,10 @@ configProvisioner:
 
   # release service configurations
   harborServerExternal: https://registry-oci.kind.internal
-  releaseServiceRootUrl: ""
+  releaseServiceRootUrl: "oci://registry-rs.edgeorchestration.intel.com"
   releaseServiceProxyRootUrl: "oci://rs-proxy.rs-proxy.svc.cluster.local:8443"
-  manifestPath: "/edge-orch/en/files/manifest"
-  manifestTag: "latest"
+  manifestPath: "/edge-orch/en/file/cluster-extension-manifest"
+  manifestTag: "v1.0.36"
```

Comment:
this is the tag that we override in EMF?

Reply:
We dont need this in EMF. this is just for running coder-rebuild/redeploy

Reply:
This will be overridden by EMF anyways.